did/utils.py

```diff
@@ -185,7 +185,7 @@ def item(text, level=0, options=None):
     if level == 1 and options.brief:
         return
     # Four space for each level, additional space for wiki format
-    indent = level * 4
+    indent = level * (4 if options.format != "md" else 2)
```

Comment:
If I'm interpreting this correctly you want to have `2` spaces for the markdown format. I suggest to drop the syntactic sugar and just make it more obvious what's the default and what' special for markdown.

```suggestion
    indent = level * 4
    if options.format == "md":
        indent = level * 2
```